### DISCUSSION TITLE
Make Indicator Type Property a Dropdown

### DISCUSSION THREAD

**Discussion by ildiavolorosso** [SFDC-Assets/DynamicFlowProgress#14] [Ideas]
Would love to see the Indicator Type property on the component a dropdown instead of plain text.

![image](https://github.com/SFDC-Assets/DynamicFlowProgress/assets/2405713/17cb17f5-b603-49a0-9ca1-090e7b384784)

**Comment by egyptguy** [comment]
I agree that would be nice, but that is not supported in LWCs used in Flows
- that is only available for other TargetConfigs (for instance Lightning
App Builder pages). There is a way to do this through additional
customization (as seen with some components on unofficialsf.com), but it’s
not worth the overhead for such a lightweight component.

Thanks for the feedback! - Mitch
